.github/workflows/cd.yml

```
@@ -0,0 +1,38 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - synchronize
+      - opened
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+
+  plugin_zip:
+    name: Create archive
+    runs-on: ubuntu-20.04
+    env:
+      OUTPUT: High DPI Continuous.zip
```

```suggestion
      OUTPUT: High-DPI-Continuous.zip
```
Avoids need to put this in quotes later.